app/serializers/disability_claim_base_serializer.rb

```diff
@@ -69,6 +69,10 @@ def waiver_submitted
     requested_decision
   end
 
+  def claim_type
+    object_data['status_type']
```

Comment:
Avoid attribute renaming if possible.  I don't have the domain knowledge here to know if `status_type`--->`claim_type` is warranted, but figured I'd poke at it